dds/DCPS/DataReaderImpl_T.h

```diff
@@ -986,6 +987,16 @@ namespace OpenDDS {
     DataReaderImpl::qos_change(qos);
   }
 
+  void set_marshal_skip_serialize(bool value)
```

Comment:
Why set/get?

Reply:
this is not something we're going to change here in the merge